controller/src/main/java/org/jboss/as/controller/OperationContext.java

```diff
@@ -700,7 +736,7 @@ default ServiceTarget getServiceTarget() throws UnsupportedOperationException {
     boolean revertResourceRestarted(PathAddress resource, Object owner);
 
     /**
-     * Resolves any expressions in the passed in ModelNode.
+     * Resolves any expressions passed in the ModelNode.
      * Expressions may either represent system properties or vaulted date. For vaulted data the format is
      * ${VAULT::vault_block::attribute_name::sharedKey}
```

Comment:
Since you are touching this...

Expressions may either represent system properties or environment variables or can be extension expressions. Extension expressions must be of the form {@code ${extensionidentifier::someextensionspecificdetails}}, where
 * {@code extensionidentifier} is some string that identifies the desired extension.

BUT

Perhaps just remove these two lines as this isn't really the place to explain expression grammar. The 'represent system properties or environment variables' bit is hardly complete, and then there are unmentioned things like nested expressions etc.